### PR TITLE
Pal 192 audit service unit tests

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

File: audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/request/ReadRequestCompleteAuditRequest.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        final ReadRequestCompleteAuditRequest that = (ReadRequestCompleteAuditRequest) o;
+        return new EqualsBuilder()
```
Comment (dev-958) [inline audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/request/ReadRequestCompleteAuditRequest.java]: Remove dependencies on EqualsBuilder() and use the intellij tool to generate the code

File: audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/request/ReadRequestCompleteAuditRequest.java [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                .append(leafResource, that.leafResource)
+                .append(rulesApplied, that.rulesApplied)
+                .append(numberOfRecordsReturned, that.numberOfRecordsReturned)
+                .append(numberOfRecordsProcessed, that.numberOfRecordsProcessed)
+                .isEquals();
+    }
+
+    @Override
+    public int hashCode() {
+        return new HashCodeBuilder(29, 37)
```
Comment (dev-958) [inline audit-service/src/main/java/uk/gov/gchq/palisade/service/audit/request/ReadRequestCompleteAuditRequest.java]: Remove dependencies on HashCodeBuilder() and use the intellij tool to generate the code

**Review by dev-958 — CHANGES_REQUESTED** [review]
Please remove all EqualsBuilder() and HashCodeBuilder() calls and use the intellij code generation tools.

File: audit-service/pom.xml [hunk 3]
```diff
@@ -82,6 +82,12 @@
             <artifactId>javax.activation-api</artifactId>
             <version>1.2.0</version>
         </dependency>
+        <dependency>
```
Comment (dev-958) [inline audit-service/pom.xml]: why are you including the palisade as a test dependency?

**Review by dev-958 — CHANGES_REQUESTED** [review]
one comment...

**Review by dev-958 — DISMISSED** [review]

**Review by dev-958 — DISMISSED** [review]
the palisade-service is still referenced in the pom

**Comment by dev930018** [comment]
Please ignore the above, should have been for the resource unit tests

**Review by nw1984 — APPROVED** [review]

**Review by m09526 — APPROVED** [review]

**Pull request merged**